actionmailer/lib/action_mailer/test_helper.rb

```diff
@@ -155,7 +155,8 @@ def assert_no_enqueued_emails(&block)
       def delivery_job_filter(job)
         job_class = job.is_a?(Hash) ? job.fetch(:job) : job.class
 
-        Base.descendants.map(&:delivery_job).include?(job_class)
+        Base.descendants.map(&:delivery_job).include?(job_class) ||
+        ActionMailer::Parameterized::DeliveryJob == job_class
```

Comment:
There is no rubocop rules for that sadly, but I think the convention is to indent multiline conditions 

```ruby
Base.descendants.map(&:delivery_job).include?(job_class) ||
  ActionMailer::Parameterized::DeliveryJob == job_class
```

Reply:
Fixed.